src/btree/bt_debug.c

```diff
@@ -1175,27 +1185,30 @@ __debug_page_metadata(WT_DBG *ds, WT_REF *ref)
     WT_SESSION_IMPL *session;
     uint64_t split_gen;
     uint32_t entries;
+    uint16_t flag_num;
```

Comment:
nit: Looking at how `flag_num` is used it could be a "first_flag" boolean instead of an integer

Reply:
I've just created a ticket that if fixed, will completely change this section of code, in this ticket they need a counter to track number of flags set so i thought i'd keep it anyways. https://jira.mongodb.org/browse/WT-12369